pom.xml

```diff
@@ -190,6 +190,19 @@
             <artifactId>cglib</artifactId>
             <version>3.3.0</version>
         </dependency>
+
+        <!-- JsonPath -->
+        <dependency>
+            <groupId>com.jayway.jsonpath</groupId>
+            <artifactId>json-path</artifactId>
+            <version>2.5.0</version>
+        </dependency>
+
+        <dependency>
```

Comment:
This "reflections" dependency seems to be an addition coming from a former resolved merge conflict and might be unused.